ts/src/config.ts

```diff
@@ -144,10 +144,17 @@ export interface Config extends GoogleAuthOptions {
   // All locations in profiles will reference locations in the running
   // JavaScript.
   disableSourceMaps?: boolean;
+
+  // Number of times to retry querying GCP metadata for the instance, zone,
+  // and projectId.
+  metadataRetries?: number;
```

Comment:
Just to make sure we use "retry" terminology properly - these are retries, so the total maximum number of attempts to talk to the metadata server is 4, right? That is, 1 initial attempt + 3 retries.

Reply:
Yep!
I added two test cases which (together) validate that the term "retries" is used correctly here.